jee-learning-platform/run_backend.py

```
@@ -0,0 +1,189 @@
+        print(f"❌ Failed to install dependencies: {e}")
+        print(f"Output: {e.stdout}")
+        print(f"Error: {e.stderr}")
+        return False
+
+def setup_database():
+    """Set up and seed the database"""
+    print("\n🗄️  Setting up database...")
+    try:
+        # Import and run the seeding script
+        sys.path.append(".")
+        import seed_data
```

Ruff (0.11.9)

`seed_data` imported but unused

Remove unused import: `seed_data`

(F401)